docs_website/docs/developer_guide/run_db_migration.md

```diff
@@ -4,12 +4,27 @@ title: DB Migration
 sidebar_label: DB Migration
 ---
 
-To make changes to the SQLAlchemy database model definitions, you need to run the following migrations:
+## Adding a new DB migration
+
+First, make changes to the SQLAlchemy database model definitions under `querybook/server/models/`,
+then you can run the following code to generate the migration file:
 
 ```sh
-# Edit querybook/alembic.ini sqlalchemy.url to point to the database
+# Edit querybook/alembic.ini sqlalchemy.url to point to the database you want to change
 # go into the docker terminal
```

Comment:
for auto generating the commit, does it also need to run inside the docker container?

Reply:
yes of course